exchange/utils.go

```diff
@@ -46,14 +46,18 @@ func cleanOpenRTBRequests(ctx context.Context,
 	privacyConfig config.Privacy,
 	account *config.Account) (allowedBidderRequests []BidderRequest, privacyLabels metrics.PrivacyLabels, errs []error) {
 
-	impsByBidder, err := splitImps(req.BidRequest.Imp)
-	if err != nil {
-		errs = []error{err}
+	aliases, errs := parseAliases(req.BidRequest)
+	if len(errs) > 0 {
 		return
 	}
 
-	aliases, errs := parseAliases(req.BidRequest)
-	if len(errs) > 0 {
+	allowedBidderRequests = make([]BidderRequest, 0, 0)
+
+	bidderToBidderResponse := processStoredBidResponses(req, aliases)
```

Comment:
I cannot move this file outside of exchange - it has dependencies that are private to exchange package

Reply:
I tried to do this. To my opinion it added more complexity to the code. If you insist and think this will make it clearer - I'm ok to add this change. Not sure if we want to integrate it to existing `stored_responses.go` file.